README.asciidoc

```diff
@@ -196,11 +196,13 @@ https://bitbucket.org/portix/dwb/pull-requests/22/several-cleanups-to-increase-p
 main inspiration for qutebrowser)
 * http://sourceforge.net/p/vimprobable/wiki/Home/[vimprobable] (C, GTK+ with
   WebKit1)
-* http://pwmt.org/projects/jumanji/[jumanji] (C, GTK+ with WebKit1)
+* http://pwmt.org/projects/jumanji/[jumanji-dead link] (C, GTK+ with WebKit1,
+https://wiki.archlinux.org/index.php?title=Jumanji[Arch Linux] has some data)
 * http://conkeror.org/[conkeror] (Javascript, Emacs-like, XULRunner/Gecko)
 * Firefox addons (not based on WebExtensions or no recent activity):
   http://www.vimperator.org/[Vimperator],
-  http://5digits.org/pentadactyl/[Pentadactyl],
+  http://5digits.org/pentadactyl/[Pentadactyl-dead link],
```

Comment:
Do we need to keep the dead link around? Unless we think it will come back there, I don't see a reason to.